chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationBody.java

```diff
@@ -48,66 +46,10 @@ public CertificationBody(final CertificationBodyEntity entity) {
         }
     }
 
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(final Long id) {
-        this.id = id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(final String name) {
-        this.name = name;
-    }
-
-    public String getWebsite() {
-        return website;
-    }
-
-    public void setWebsite(final String website) {
-        this.website = website;
-    }
-
-    public Address getAddress() {
-        return address;
-    }
-
-    public void setAddress(final Address address) {
-        this.address = address;
-    }
-
-    public String getAcbCode() {
-        return acbCode;
-    }
-
-    public void setAcbCode(final String acbCode) {
-        this.acbCode = acbCode;
-    }
-
-    public boolean isRetired() {
-        return retired;
-    }
-
-    public void setRetired(final boolean retired) {
-        this.retired = retired;
-    }
-
-    public Date getRetirementDate() {
-        return retirementDate;
-    }
-
-    public void setRetirementDate(final Date retirementDate) {
-        this.retirementDate = retirementDate;
-    }
 
     @Override
     public String toString() {
```

Comment:
Do we need a custom `toString`? If we're converting the getters/setters to Lombok versions, can we do the same with the toString?